util/src/types.rs

```diff
@@ -12,64 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-//! Logging configuration types
-
-/// Log level types
-#[derive(Debug, Clone, Serialize, Deserialize, PartialEq)]
-pub enum LogLevel {
-	/// Error
-	Error,
-	/// Warning
-	Warning,
-	/// Info
-	Info,
-	/// Debug
-	Debug,
-	/// Trace
-	Trace,
-}
-
-/// 32 log files to rotate over by default
-pub const DEFAULT_ROTATE_LOG_FILES: u32 = 32 as u32;
-
-/// Logging config
-#[derive(Debug, Clone, Serialize, Deserialize, PartialEq)]
-pub struct LoggingConfig {
-	/// whether to log to stdout
-	pub log_to_stdout: bool,
-	/// logging level for stdout
-	pub stdout_log_level: LogLevel,
-	/// whether to log to file
-	pub log_to_file: bool,
-	/// log file level
-	pub file_log_level: LogLevel,
-	/// Log file path
-	pub log_file_path: String,
-	/// Whether to append to log or replace
-	pub log_file_append: bool,
-	/// Size of the log in bytes to rotate over (optional)
-	pub log_max_size: Option<u64>,
-	/// Number of the log files to rotate over (optional)
-	pub log_max_files: Option<u32>,
-	/// Whether the tui is running (optional)
-	pub tui_running: Option<bool>,
-}
-
-impl Default for LoggingConfig {
-	fn default() -> LoggingConfig {
-		LoggingConfig {
-			log_to_stdout: true,
-			stdout_log_level: LogLevel::Warning,
-			log_to_file: true,
-			file_log_level: LogLevel::Info,
-			log_file_path: String::from("grin.log"),
-			log_file_append: true,
-			log_max_size: Some(1024 * 1024 * 16), // 16 megabytes default
-			log_max_files: Some(DEFAULT_ROTATE_LOG_FILES),
-			tui_running: None,
-		}
-	}
-}
+//! Zeroing String
```

Comment:
Weird comment here?

Reply:
It's module documentation and it seems to be required for all public modules. Without it we get a warning due to the #![warn(missing_docs)] in lib.rs.

After I removed the logging code from this file I just updated the comment to say ZeroingString as that is all that is left in there. Maybe the file should be renamed to zeroing_string.rs. Or a better comment?

Reply:
Okay let's leave it like that for now.